### PR TITLE
Build: update Dockerfile.gnu with mpifort for paw test case

### PR DESCRIPTION
**Pull request by hongriTianqi**
This is to fix the CI error happened due to absence of mpifort for 101_PW_15_paw test case

### CODE REVIEW

**Comment by hongriTianqi** [comment]
@caic99 The change of docker file seems can not solve the problem. How can we update the docker image for running the CI tests?

File: Dockerfile.gnu [hunk 1]
```diff
@@ -19,6 +19,10 @@ ENV CMAKE_PREFIX_PATH=/opt/libtorch/share/cmake
 
 ADD https://api.github.com/repos/deepmodeling/abacus-develop/git/refs/heads/develop /dev/null
 
+RUN apt-get install -y libmpich-dev mpich
```
Comment (caic99) [inline Dockerfile.gnu]: Please put `apt-get` commands all together.

**Review by caic99 — CHANGES_REQUESTED** [review]

**Comment by caic99** [comment]
> @caic99 The change of docker file seems can not solve the problem. How can we update the docker image for running the CI tests?

@hongriTianqi Once this PR is merged, the new image will be built and applies to the following tests.

**Review by caic99 — APPROVED** [review]

**Review by dyzheng — APPROVED** [review]
This PR seems can solve the CI test error for PAW cases.
Only GNU image has been fixed in this PR.

**Comment by dyzheng** [comment]
Can you link the PR which caused this problem?

**Comment by caic99** [comment]
@hongriTianqi I would suggest reverting this PR since it does not actually solves the problem which has been fixed by @wenfei-li .

**Pull request merged**